frontend/public/components/nav/_nav-header.scss

```diff
@@ -1,11 +1,6 @@
 .oc-nav-header {
   border-bottom: 1px solid $color-pf-black;
-  padding: var(--pf-global--spacer--md) var(--pf-global--spacer--sm);
-
-  @media screen and (min-width: $grid-float-breakpoint) {
-    padding-left: var(--pf-global--spacer--md);
-    padding-right: var(--pf-global--spacer--md);
-  }
+  padding: var(--pf-c-nav__list-link--PaddingTop) var(--pf-c-nav__list-link--PaddingRight) var(--pf-c-nav__list-link--PaddingTop) var(--pf-global--spacer--md);
```

Comment:
I think we want to keep the values as they were as we need the extra pixels at mobile and the heading contents and page contents are misaligned at mobile.